### PR TITLE
Feature/71 display photo of the day

### PR DESCRIPTION
**Pull request by ostartt**

### CODE REVIEW

**Review by NastiaChaikovska — CHANGES_REQUESTED** [review]
do git pull before merge

File: src/pages/home/home-page.css [hunk 1]
```diff
@@ -45,8 +45,9 @@
 }
 
 .home-page main{
-    width: 80%;
```
Comment (NastiaChaikovska) [inline src/pages/home/home-page.css:48]: why width in px

**Review by NastiaChaikovska — CHANGES_REQUESTED** [review]
most-popular-and-commented-section should be only once

File: src/pages/home/home-page.css (hunk 1 above)
Comment (ostartt) [reply src/pages/home/home-page.css:48]: it's temporary)

**Review by NastiaChaikovska — APPROVED** [review]

**Pull request merged**